odoo/addons/test_main_flows/tests/test_flow.py

```diff
@@ -109,9 +109,8 @@ def test_company_switch_access_error(self):
             "action": f"ir.actions.act_window,{act_window.id}",
         })
 
-        current_companies = "%s-%s" % (company1.id, company2.id)
         with mute_logger("odoo.http"):
-            self.start_tour(f"/web#action={act_window.id}&cids={current_companies}", "test_company_switch_access_error", login="admin")
+            self.start_tour(f"/web#action={act_window.id}", "test_company_switch_access_error", login="admin", cookies={"cids": f"{company1.id}-{company2.id}"})
```

Comment:
Note that using cookies will probably quickly lead to having all of them in cookies / a cookie mess. At least URLs were clear about a specific link using some CID